plio/utils/covariance.py

```diff
@@ -3,13 +3,15 @@
 
 def compute_covariance(lat, lon, rad, latsigma=10., lonsigma=10., radsigma=15., semimajor_axis=None):
     """
-    Given geospatial coordinates, desired accuracy sigmas, and an equitorial radius, compute a 2x3
-    sigma covariange matrix.
+    Given geospatial coordinates, desired accuracy sigmas, and an equitorial radius, computes the rectangular
```

Comment:
Mention that they are "ground distance sigmas" because it can be confusing if they are distance sigmas or angular sigmas.

Reply:
Probably should also add this to the specific parameter docs. Something like "The uncertainty in the latitude direction in meters".